elastalert/alerts.py

```diff
@@ -257,6 +257,10 @@ def get_aggregation_summary_text__maximum_width(self):
 
     def get_aggregation_summary_text(self, matches):
         text = ''
+        summary_prefix = None
+        if 'summary_prefix' in self.rule and self.rule['summary_prefix']:
+            summary_prefix = self.rule['summary_prefix']
+            text = '{{{0}}}'.format(summary_prefix)
```

Comment:
I don't think we should include `{}` automatically. If brackets are needed, the user would add them into the summary_prefix option.

Reply:
Sounds reasonable, I will change it.